nrf_802154/sl/sl_opensource/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+add_library(nrf-802154-sl STATIC)
+
+target_link_libraries(nrf-802154-sl PUBLIC nrf-802154-platform)
+
+target_include_directories(nrf-802154-sl
+  PUBLIC
+    ../include
+    .
+)
+
+target_sources(nrf-802154-sl
+  PRIVATE
+    nrf_802154_sl_ant_div.c
```

Comment:
Could we add `src` and `include` subdirectories here for consistency with other libraries?

Reply:
Okay, it can be done.